.github/workflows/deploy-cloud-nodes.yml

```diff
@@ -4,21 +4,21 @@ on:
   workflow_dispatch:
     inputs:
       build-image:
-        description: 'Build docker image'
+        description: "Build docker image"
         required: false
-        default: 'true'
+        default: "true"
       deploy-iris:
-        description: 'Deploy Iris'
+        description: "Deploy Iris"
         required: false
-        default: 'true'
+        default: "true"
       deploy-anthony:
-        description: 'Deploy Anthony'
+        description: "Deploy Anthony"
         required: false
-        default: 'true'
+        default: "true"
       deploy-brad:
-        description: 'Deploy Brad'
+        description: "Deploy Brad"
         required: false
-        default: 'true'
+        default: "true"
```

Comment:
How can we avoid all of these changes between single quote and double quote characters? Seems like a conflict in linters

Reply:
I think this is my bad, I think I have an extension that is auto-formatting `yml` files. Normally I work around it by not triggering the linter but sometimes I do it accidently.

I'll track down what extension is doing that and remove it.